src/Components/Create/Create.jsx

```diff
@@ -126,92 +151,264 @@ class Create extends React.Component {
+                                                    name='P2'
+                                                    value='ME'
+                                                    checked={this.state.P2 === "ME"}
+                                                    onChange={this.handleOptionChange}
+                                                    className='radio-input'
+                                                />
+                                                Me, myself and I
+                                            </label>
+                                        </div>
+                                        <div>
+                                            <label>
+                                                <input
+                                                    type='radio'
+                                                    name='P2'
+                                                    value='AI'
+                                                    checked={this.state.P2 === "AI"}
+                                                    onChange={this.handleOptionChange}
+                                                    className='radio-input'
+                                                />
+                                                Deeper Blue
+                                            </label>
+                                        </div>
+                                        <div>
+                                            <label>
+                                                <input
+                                                    type='radio'
+                                                    name='P2'
+                                                    value='OPEN'
+                                                    checked={this.state.P2 === "OPEN"}
+                                                    onChange={this.handleOptionChange}
+                                                    className='radio-input'
+                                                />
+                                                Open slot
+                                            </label>
+                                        </div>
+                                    </div>
+                                    <div className='time-choice'>
+                                        <div className='time-header'>
+                                            Time Control
+                                        </div>
+                                        <div>
+                                            <label>
+                                                <input
+                                                    type='radio'
+                                                    name='P2time'
+                                                    value='0'
+                                                    checked={this.state.P2time === "0"}
```

Comment:
Set this to a really high value instead of 0 because the server treats that as game over

Reply:
ok